tests/php/Integration/TestTemplateOutput.php

```diff
@@ -148,14 +148,14 @@ public function test_block_template() {
 		$actual_template   = str_replace( [ "\t", "\n" ], '', $rendered_template );
 
 		// The 'className' should be present.
-		$this->assertContains(
+		$this->assertStringContainsString(
```

Comment:
`assertContains()` causes a PHPUnit error in PHP 8.1, as a string is not iterable